### PR TITLE
BZ-1707236: Adding step to create a secret for installing ansible SB

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1707236/applications/service_brokers/installing-ansible-service-broker.html

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@shawn-hurley, @johnkim76 if one of you could review these changes please, thanks!

File: modules/sb-install-asb-operator.adoc [hunk 1]
```diff
@@ -27,6 +27,13 @@ NOTE: The namespace must start with `openshift-`.
  `openshift-ansible-service-broker` namespace, and enter `openshift-ansible-service-broker-operator` in the
  *Subject Name* field.
 .. Click *Create*.
+. Create a secret to connect to the Red Hat Container Catalog.
+.. Navigate to *Workloads* -> *Secrets*. Verify that the `openshift-ansible-service-broker` project is selected.
+.. Click *Create* -> *Key/Value Secret*.
+.. Enter `asb-auth-secret` as the *Secret Name*.
```
Comment (zihantang-rh) [inline modules/sb-install-asb-operator.adoc]: The secret should be named as `asb-registry-auth`

File: modules/sb-start-asb.adoc [hunk 2]
```diff
@@ -19,8 +19,20 @@ using the following procedure.
 // TODO: Change to *OpenShift Ansible Service Broker Operator* by GA (2 of 5)
 . Select the *Automation Broker Operator*.
 . Under *Provided APIs*, click *Create New* for *Automation Broker*.
-. Review the default YAML, set any additional {asb-name} configuration options,
-and click *Create*.
+. Add the following to the `spec` field in the default YAML provided:
```
Comment (zihantang-rh) [inline modules/sb-start-asb.adoc]: Don't need to add registry in spec, can use the default directly, by default, the registry is:
``` 
 - auth_name: asb-registry-auth
    auth_type: secret
    name: rhcc
    type: rhcc
    url: https://registry.redhat.io
    white_list:
    - .*-apb$
```

Comment (zihantang-rh) [reply modules/sb-start-asb.adoc]: this secret is created in last step.

File: modules/sb-start-asb.adoc [hunk 3]
```diff
@@ -19,8 +19,20 @@ using the following procedure.
...
+[source,yaml]
+----
+registry:
+  - name: rhcc
+    type: rhcc
+    url: registry.access.redhat.com
+    auth_type: secret
+    auth_name: asb-auth-secret
+----
++
```
Comment (zihantang-rh) [inline modules/sb-start-asb.adoc]: `registry.access.redhat.com` is used as default before v3.10, after v3.11+, the default changed to `registry.redhat.io`

If using `registry.access.redhat.com`, do NOT need secret, like following:
```
      - type: rhcc
        name: rh
        url:  https://registry.access.redhat.com
        org:  
        white_list: [.*-apb$]
        auth_type: ""
        auth_name: ""
```

File: modules/sb-start-asb.adoc (hunk 2 above)
Comment (shawn-hurley) [reply modules/sb-start-asb.adoc]: We should keep this to avoid confusion IMO.

**Comment by shawn-hurley** [comment]
LGTM after the changes already mentioned.

**Comment by bergerhoffer** [comment]
Thanks @zihantang-rh for reviewing. I changed the name of the secret and the URL for the registry. Based on @shawn-hurley's comments, I kept the YAML there to be clearer to our users. Let me know if this looks okay now.

**Comment by zihantang-rh** [comment]
LGTM, thanks.

**Pull request merged**